lib/solveEquation/EquationOperations.js

```diff
@@ -45,12 +45,20 @@ EquationOperations.ensureSymbolInLeftNode = function(equation, symbolName) {
   return EquationStatus.noChange(equation);
 };
 
-// TODO: Ensures that a symbol is not in the denominator by multiplying
-// both sides by the whatever order of the symbol necessary.
-// This is blocked on the simplifying functionality of canceling symbols in
-// fractions (needs factoring for full canceling support)
-EquationOperations.removeSymbolFromDenominator = function(equation) {
-  // pass for now
+// Ensures that a symbol is not in the denominator by multiplying
+// both sides by the denominator if there is a symbol present.
+EquationOperations.removeSymbolFromDenominator = function(equation, symbolName) {
+  // Can't multiply a symbol across non-equal comparators
+  // because you don't know if it's negative and need to flip the sign
+  if (equation.comparator !== '=') {
+    return EquationStatus.noChange(equation);
+  }
+  const leftNode = equation.leftNode;
+  const denominator = Symbols.getLastDenominatorWithSymbolTerm(leftNode, symbolName);
+  if (denominator) {
+    return performTermOperationOnEquation(
```

Comment:
I love how simple this change is thanks to the great work @aliang8 and @ldworkin have done 😍